test/remote/gateways/remote_forte_test.rb

```diff
@@ -23,6 +23,9 @@ def setup
       billing_address: address,
       description: 'Store Purchase'
     }
+
+    @options_with_order_id = @options
+    @options_with_order_id.update(order_id:'2')
```

Comment:
I wasn't sure if we try to only have the bare minimums in the options hash. Definitely can add to all though.